kong/db/schema/typedefs.lua

```diff
@@ -456,6 +456,11 @@ typedefs.tags = Schema.define {
   description = "A set of strings representing tags."
 }
 
+typedefs.capability = Schema.define {
+  type = "string",
+  description = "A string representing an RPC capability."
+}
```

Comment:
Do we want to make this an `enumeration`?

Reply:
I considered this during development and decided against it out of worry this will create compatibility headache later for us. It is okay if DP reports capability the CP doesn't know about. If CP does not call an unknown capability then there is no side effect anyway. That's the whole reason why capabilities are advertised, not negotiated.